app/views/games/categories/leaderboards/sum_of_bests/index.slim

```diff
@@ -3,36 +3,38 @@
   h2 Fastest sum of best
```

Comment:
Breadcrumbs seem to be missing on this page.

Also, should it have the srl logo button and edit button on this page or nah?